react/AppTile/index.jsx

```diff
@@ -88,16 +88,14 @@ export const AppTile = ({
         )}
       </TileIcon>
       <TileDescription className={styles[`AppTile-description`]}>
-        <TileTitle
-          className={statusLabel ? '' : styles['AppTile-title-2lines']}
-        >
+        <TileTitle isMultiline={!statusLabel}>
           {namePrefix ? `${namePrefix} ${name}` : name}
         </TileTitle>
         {developer.name && showDeveloper && (
           <TileSubtitle>{`${t('app_item.by')} ${developer.name}`}</TileSubtitle>
         )}
         {statusToDisplay && (
-          <TileFooter className={styles[`AppTile-footer-${statusLabel}`]}>
+          <TileFooter isAccent={statusLabel === 'update'}>
```

Comment:
Same, a pseudo enum could be nice

Reply:
Should be better : https://github.com/cozy/cozy-ui/pull/1885/commits/7e38c50800174a3ef738f9e5c11efe2e0970e6e3